Tests/NIOPosixTests/EventLoopTest+XCTest.swift

```diff
@@ -2,7 +2,7 @@
 //
 // This source file is part of the SwiftNIO open source project
 //
-// Copyright (c) 2017-2021 Apple Inc. and the SwiftNIO project authors
+// Copyright (c) 2017-2022 Apple Inc. and the SwiftNIO project authors
```

Comment:
This is currently breaking on the CI and was caused by my previous PR where the CI did not detect this necessary change. I am just going to include it here to fix the CI.